contrib/executor/cypress/pkg/runner/cypress.go

```diff
@@ -265,6 +265,10 @@ func MapStatus(in junit.Status) (out string) {
 	switch string(in) {
 	case "passed":
 		return string(testkube.PASSED_ExecutionStatus)
+	case "pended":
```

Comment:
should this be pending?
```suggestion
	case "pending":
```

Reply:
good question, checking with user)